submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/SubmissionRepository.kt

```diff
@@ -22,4 +24,13 @@ class SubmissionRepository(
             submissionRepository.save(it)
         }
     }
+
+    fun findSubmissionsByAccessTags(accessTags: List<AccessTag>) =
+        submissionRepository.findDistinctByRootSectionTypeAndAccessTagsInAndVersionGreaterThan(
+            Companion.PROJECT_TYPE, accessTags, 0).map { submissionDbMapper.toSubmission(it) }
+
+    companion object {
+        const val PROJECT_TYPE = "Project" //TODO: refactor to enum when submission type is handled
```

Comment:
You can just define the constant outside the class to avoid the `companion` part

Reply:
Was avoiding a naming convention error but let's just refactor it to introduce submission type right now as it would be used later anyway